### DISCUSSION TITLE
Utility meter with Monty Cycle to reset second week of the month

### DISCUSSION THREAD

**Discussion by fibilt** [BottlecapDave/HomeAssistant-OctopusEnergy#146] [Q&A]
I wonder if you have been able to set up a monthly utility meter using the electricity and gas sensors in kWh from this integration? I might be doing something wrong, but I cant get this to work.

` 
um_daily_electricity:
    source: sensor.octopus_energy_electricity_17p2225658_1200025442284_previous_accumulative_consumption
    name: Daily Electricity Consumption
    unique_id: um_daily_electricity
    cycle: daily

  um_monthly_electricity:
    source: sensor.octopus_energy_electricity_17p2225658_1200025442284_previous_accumulative_consumption
    name: Monthly Electricity Consumption
    unique_id: um_monthly_electricity
    cycle: monthly
    offset:
      days: 6

  um_daily_gas:
    source: sensor.octopus_energy_gas_g4p21843641700_621077107_previous_accumulative_consumption_kwh
    name: Daily Gas Consumption
    unique_id: um_daily_gas
    cycle: daily

  um_monthly_gas:
    source: sensor.octopus_energy_gas_g4p21843641700_621077107_previous_accumulative_consumption_kwh
    name: Monthly Gas Consumption
    unique_id: um_monthly_gas
    cycle: monthly
    offset:
      days: 6
`

Any help or pointer to the right direction is appreciated. Thanks

**Comment by BottlecapDave** [comment]
I haven't really used this integration. Did you solve your issue in the end?
